### ISSUE TITLE
[Question] how to store images locally

### ISSUE THREAD

**Issue by dougcooper** [0xGG/crossnote-app#144]
when i upload an image it gets stored on a remote server. how do i show local images?

**Comment by shd101wyy** [comment]
Right now it is not possible to be achieved in crossnote web.    
You will need to do that locally on your computer.

**Comment by dougcooper** [comment]
When you say locally do you mean using the vscode plugin?

**Comment by shd101wyy** [comment]
Not necessarily. You can add images locally, then do a git push locally. After that, you pull that repo in Crossnote web, and you will be able to embed local images like `![](./test.png)`. I currently doesn't support store images in Crossnote web directly, but the feature will definitely be supported in the future.

**Comment by dongshunyao** [comment]
The project is being refactored, so we will close outdated issues. Once the new version is released, please open a new issue if you encounter any problems. Thank you for your contributions and support!
